web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -141,14 +137,17 @@ const TeamReviews = ({ onBack, periodId }) => {
   const history = useHistory();
   const location = useLocation();
 
+  const [assignments, setAssignments] = useState([]);
```

Comment:
This holds all the assignments (member <-> reviewer) for the current review period.